policy-controller/grpc/src/lib.rs

```diff
@@ -369,20 +369,20 @@ fn to_http_route_list<'r>(
     // comparison, because all these routes will exist in the same
     // namespace.
     let mut route_list = routes.into_iter().collect::<Vec<_>>();
-    route_list.sort_by(|(a_name, a), (b_name, b)| {
+    route_list.sort_by(|(a_ref, a), (b_ref, b)| {
         a.creation_timestamp
             .cmp(&b.creation_timestamp)
```

Comment:
Practically, it's not a concern: we should never be mixing default routes and api-defined routes. It's worth a comment describing that.

Practically, all default routes should have the same timestamp. I agree, however, that we shouldn't default and defined route refs by name as equivalent. My suggestion would be to implement `PartialOrd for RouteRef` such that Default routes are always greater than resource-defined routes so that resource-defined routes are always preferred. If we follow my other suggestion of using `UNIX_EPOCH` for the timestamp, we'll ensure this, as well.